package.json

```diff
@@ -134,7 +134,8 @@
     "qs": "^6.11.2",
     "redis": "^3.1.1",
     "semver": "^7.5.4",
-    "tough-cookie": "^4.1.3"
+    "tough-cookie": "^4.1.3",
+    "browserify-sign": "^4.2.2"
```

Comment:
how come we're adding a new dependency? or is it a requirement of the new version of tough-cookie